src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/DSDropDownBase.cs

```diff
@@ -36,18 +36,17 @@ public void CustomizeView(DSDropDownBase model, NodeView nodeView)
             combo.SelectionChanged += delegate
             {
                 if (combo.SelectedIndex != -1)
+                {
                     model.OnNodeModified();
```

Comment:
Can you try out your code changes with both `Run Automatic` and `Run Manually` modes? When you do, please add a comment in the defect by getting TDs to try both scenarios out also.